src/Ch9/Ch9.UWP/Ch9.UWP.csproj

```diff
@@ -115,6 +115,9 @@
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Toolkit.Uwp.UI.Lottie">
+      <Version>6.1.0</Version>
```

Comment:
You could put the version on the same line like in the other references

Reply:
Yep. Nuget?